src/test/lib/operators/sort_test.cpp

```diff
@@ -116,7 +116,7 @@ inline std::string sort_test_formatter(const testing::TestParamInfo<SortTestPara
   }
   stream << (param.input_is_reference ? "Reference" : "Data") << "Input";
   for (const auto& sort_column : param.sort_columns) {
-    stream << "Col" << sort_column.column << sort_mode_to_string.left.at(sort_column.sort_mode);
```

Comment:
Not yours, but that looks ugly. How does not look when printed? `coll_orderkeyAscending`?

Reply:
Hmm, no. But then again: it's just autogenerated test names. Feel free to resolve.
Sorry.